CMakeLists.txt

```diff
@@ -307,7 +312,9 @@ set_target_properties(lib${APP_NAME_LC} PROPERTIES PREFIX "")
 add_dependencies(lib${APP_NAME_LC} libcpluff ffmpeg dvdnav crossguid)
 
 # Codegen
-add_subdirectory(${CMAKE_SOURCE_DIR}/xbmc/interfaces/swig build/swig)
+if(ENABLE_PYTHON)
```

Comment:
The only thing I could imagine is that it either needs some vars from the main cmakelists or would get introduced too late otherwise. Maybe you can just quickly try if it compiles, otherwise we can also leave it here. Maybe you had it explicitly there for a reason: https://github.com/fetzerch/xbmc/commit/538ef57c8b78b85ac592fa9caa1a55f7d991378d :D